deposit/src/main/java/edu/unc/lib/deposit/fcrepo3/IngestDeposit.java

```diff
@@ -371,9 +371,7 @@ private void ingestObject(String ingestPid, boolean confirmExisting) throws Depo
 		uploadIngestFiles(foxmlDoc, pid);
 
 		// Ingest the object's FOXML
-		try {
-
-			ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+		try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
```

Comment:
In this case I left it as ByteArrayOutputStream because it needs to call outputStream.toByteArray() at line 382, which is specific to ByteArrayOutputStream